vsintegration/src/FSharp.Editor/CodeFix/CodeFixHelpers.fs

```diff
@@ -19,15 +19,12 @@ module internal CodeFixHelpers =
             computeTextChanges: unit -> Async<TextChange[] option>
         ) =
 
-        // I don't understand how we can get anything but a single diagnostic here - as we get a single "title" here.
-        // But since we don't have proper testing yet, keeping it like this to verify this theory in telemetry.
-        let ids = context.Diagnostics |> Seq.map (fun d -> d.Id) |> String.concat ","
+        let ids = context.Diagnostics |> Seq.exactlyOne
```

Comment:
Per Roslyn documentation, codefixes can have multiple diagnostics, in case there are multiple for the same range.
Therefore it is possible to have > 1 in that collection.

exactlyOne would cause it to fail

No telementry-collection code should ever cause regular code to fail, I think this is a property we must maintain all the time.

Reply:
@psfinaki , can we close this, since it is currently correct, or do we want it to start failing if multiple diagnostics start occuring?

Reply:
@T-Gro @KevinRansom so I reverted that change, just clarified the comment. 
The "title" still should be IMO removed for the GDPR purposes.